CONTRIBUTING.md

```diff
@@ -1,79 +1,232 @@
-# Contributing to AsyncAPI
-We love your input! We want to make contributing to this project as easy and transparent as possible.
+# AsyncAPI Extensions Catalog Contribution Guide
 
-## Contribution recogniton
+AsyncAPI is an evolving language. This repository contains the Extensions as well as Pull Requests with suggested improvements and contributions.
 
-We use [All Contributors](https://allcontributors.org/docs/en/specification) specification to handle recognitions. For more details read [this](https://github.com/asyncapi/community/blob/master/recognize-contributors.md) document.
+Contributions that do not change the interpretation of the extensions but instead improve legibility, fix editorial errors, clear up ambiguity and improve examples are encouraged and are often merged by a extension Committer with little process.
 
-## Summary of the contribution flow
+However, contributions that _do_ meaningfully change the interpretation of the extensions must follow an RFC (Request For Comments) process through a series of *stages* intended to improve *visibility*, allow for *discussion* to reach the best solution, and arrive at *consensus*. This process becomes even more important as AsyncAPI's community broadens.
```

Comment:
We can simplify even more and copy less from the spec. So remove these 3 paragraphs and write something like

```
Extensions catalog is like a waiting room for main specification features. Goal for the extensions catalog is to enable easier and faster contribution process than you see in [AsyncAPI Specification](https://github.com/asyncapi/spec/blob/master/CONTRIBUTING.md). 
```

Reply:
I didn't understand this clearly

Reply:
instead of 

![Screenshot 2023-09-26 at 11 20 21](https://github.com/asyncapi/extensions-catalog/assets/6995927/338d3a9e-68fc-4403-b6ed-e50f2e0b3715)

write something similar to what I suggested